HW_4/task_3.js

```diff
@@ -0,0 +1,52 @@
+/*
+5*. Напишите рекурсивную функцию, которая принимает на вход число и складывает его цифры. 
+  Если сумма получилась больше 9 - снова сложите цифры. И так пока, сумма не станет меньше либо равной 9. 
+  После окончания сложений возвращает полученное число. Например при подаче числа 19 (1+9=10>9, потому 1+0=1) выводится 1
+
+6*. Написать функцию, которая принимает на вход строку с текстом, и заменяет каждую пару стоящих подряд идентичных букв на одну следующую в алфавите, 
+    и так пока в тексте не останется двух одинаковых букв стоящих рядом (через пробел и другой знак препинания можно)
+    Пример: aabc => bbc => cc => d
+
+*/
+function reсursionSumNumbers(num){
+    if(num < 10){
+        return num;
+    }
+    let sum = 0;
+    for(let i of String(num).split('')){
+        sum += Number(i);
+    }
+    return reсursionSumNumbers(sum)
+}
+console.log(reсursionSumNumbers(912345))
+
+
+function recursionEqualSymbols(str) {

```

Comment:
Solution works correctly, but try to solve it with less lines (not necessary)
As main condition u can use `str[0] !== str[1]` or `str.length > 1`, some .replace, .replaceAll, .slice can come in handy